itou/eligibility/models/iae.py

```diff
@@ -290,42 +232,44 @@ class AdministrativeCriteria(models.Model):
 
     MAX_UI_RANK = 32767
 
-    class Level(models.TextChoices):
-        LEVEL_1 = "1", "Niveau 1"
-        LEVEL_2 = "2", "Niveau 2"
+    level = models.CharField(
+        verbose_name="Niveau",
+        max_length=1,
+        choices=AdministrativeCriteriaLevel.choices,
+        default=AdministrativeCriteriaLevel.LEVEL_1,
+    )
 
-    level = models.CharField(verbose_name="Niveau", max_length=1, choices=Level.choices, default=Level.LEVEL_1)
     name = models.CharField(verbose_name="Nom", max_length=255)
     desc = models.CharField(verbose_name="Description", max_length=255, blank=True)
     written_proof = models.CharField(verbose_name="Justificatif", max_length=255, blank=True)
     written_proof_url = models.URLField(
         verbose_name="Lien d'aide à propos du justificatif", max_length=200, blank=True
     )
     written_proof_validity = models.CharField(
-        verbose_name="Durée de validité du justificatif", max_length=255, blank=True, default=""
+        verbose_name="Durée de validité du justificatif",
+        max_length=255,
+        blank=True,
+        default="",
     )
     # Used to rank criteria in UI. Should be set by level (LEVEL_1: 1, 2, 3… LEVEL_2: 1, 2, 3…).
     # Default value is MAX_UI_RANK so that it's pushed at the end if `ui_rank` is forgotten.
     ui_rank = models.PositiveSmallIntegerField(default=MAX_UI_RANK)
     created_at = models.DateTimeField(verbose_name="Date de création", default=timezone.now)
     created_by = models.ForeignKey(
-        settings.AUTH_USER_MODEL, verbose_name="Créé par", null=True, blank=True, on_delete=models.SET_NULL
+        settings.AUTH_USER_MODEL,
+        verbose_name="Créé par",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
```

Comment:
Cette ligne n’a pas besoin de changer.
```suggestion
        on_delete=models.SET_NULL
```